utils/mel.py

```diff
@@ -231,9 +231,6 @@ def extract_mel_features_tts(
 
         spec = torch.matmul(mel_basis[str(cfg.fmax) + "_" + str(y.device)], spec)
         spec = spectral_normalize_torch(spec)
-        spec = spec.squeeze(0)
```

Comment:
Are you sure it works to delete the code in line 234, which is not a repeated code?

Reply:
Yes, it's same as lines 165 to 168.